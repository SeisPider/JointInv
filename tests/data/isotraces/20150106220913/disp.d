SURF96 R U T  0          85       3.546      0.1866 
SURF96 R U T  0          80       3.551      0.1761 
SURF96 R U T  0          75       3.555      0.1655 
SURF96 R U T  0          70       3.554      0.1544 
SURF96 R U T  0          65        3.54      0.1422 
SURF96 R U T  0          60       3.477      0.1267 
SURF96 R U T  0          55        3.46       0.115 
SURF96 R U T  0          50       3.564      0.1109 
SURF96 R U T  0          48       3.585      0.1077 
SURF96 R U T  0          46       3.563       0.102 
SURF96 R U T  0          44       3.515     0.09494 
SURF96 R U T  0          42       3.456     0.08759 
SURF96 R U T  0          40       3.403     0.08086 
SURF96 R U T  0          38       3.362     0.07498 
SURF96 R U T  0          36        3.32     0.06931 
SURF96 R U T  0          34       3.273     0.06361 
SURF96 R U T  0          32       3.207     0.05748 
SURF96 R U T  0          30       3.148      0.0519 
SURF96 R U T  0          29        3.13      0.0496 
SURF96 R U T  0          28       3.117     0.04751 
SURF96 R U T  0          27       3.109     0.04556 
SURF96 R U T  0          26       3.101     0.04366 
SURF96 R U T  0          25       3.096     0.04184 
