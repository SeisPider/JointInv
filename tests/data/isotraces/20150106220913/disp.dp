SURF96 R U T  0       81.07       3.546      0.1866 
SURF96 R U T  0       77.95       3.551      0.1761 
SURF96 R U T  0       74.53       3.555      0.1655 
SURF96 R U T  0       70.98       3.554      0.1544 
SURF96 R U T  0       67.48        3.54      0.1422 
SURF96 R U T  0       63.35       3.477      0.1267 
SURF96 R U T  0       56.92        3.46       0.115 
SURF96 R U T  0       47.04       3.564      0.1109 
SURF96 R U T  0       44.81       3.585      0.1077 
SURF96 R U T  0       43.07       3.563       0.102 
SURF96 R U T  0        41.3       3.515     0.09494 
SURF96 R U T  0       39.63       3.456     0.08759 
SURF96 R U T  0       38.11       3.403     0.08086 
SURF96 R U T  0       36.65       3.362     0.07498 
SURF96 R U T  0        35.2        3.32     0.06931 
SURF96 R U T  0        33.7       3.273     0.06361 
SURF96 R U T  0       31.85       3.207     0.05748 
SURF96 R U T  0       29.81       3.148      0.0519 
SURF96 R U T  0       29.07        3.13      0.0496 
SURF96 R U T  0       28.47       3.117     0.04751 
SURF96 R U T  0       27.95       3.109     0.04556 
SURF96 R U T  0       27.52       3.101     0.04366 
SURF96 R U T  0       27.15       3.096     0.04184 
