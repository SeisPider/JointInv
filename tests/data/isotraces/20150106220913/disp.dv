MFT96 R U T  0 85.333    3.4797    1.0000    5727.0     0.6857E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 83.592    3.4884    1.0000    5727.0     0.7487E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 81.920    3.4967    1.0000    5727.0     0.8086E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 80.314    3.5045    1.0000    5727.0     0.8644E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 78.769    3.5119    1.0000    5727.0     0.9149E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 77.283    3.5188    1.0000    5727.0     0.9593E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 75.852    3.5253    1.0000    5727.0     0.9965E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 74.473    3.5314    1.0000    5727.0     0.1026E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 73.143    3.5371    1.0000    5727.0     0.1047E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 71.860    3.5423    1.0000    5727.0     0.1059E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 70.621    3.5471    1.0000    5727.0     0.1062E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 69.424    3.5515    1.0000    5727.0     0.1055E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 68.267    3.5554    1.0000    5727.0     0.1040E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 67.148    3.5590    1.0000    5727.0     0.1015E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 66.065    3.5621    1.0000    5727.0     0.9825E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 65.016    3.5649    1.0000    5727.0     0.9416E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 64.000    3.5672    1.0000    5727.0     0.8936E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 63.015    3.5692    1.0000    5727.0     0.8393E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 62.061    3.5707    1.0000    5727.0     0.7800E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 61.134    3.5719    1.0000    5727.0     0.7171E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 60.235    3.5727    1.0000    5727.0     0.6520E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 59.362    3.5731    1.0000    5727.0     0.5869E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 58.514    3.5732    1.0000    5727.0     0.5240E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 57.690    3.5729    1.0000    5727.0     0.4664E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 56.889    3.5723    1.0000    5727.0     0.4175E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 56.110    3.5713    1.0000    5727.0     0.3812E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 55.351    3.5700    1.0000    5727.0     0.3608E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 54.613    3.5683    1.0000    5727.0     0.3575E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 53.895    3.5663    1.0000    5727.0     0.3700E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 53.195    3.5640    1.0000    5727.0     0.3945E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 52.513    3.5614    1.0000    5727.0     0.4272E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 51.848    3.5585    1.0000    5727.0     0.4649E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 51.200    3.5554    1.0000    5727.0     0.5060E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 50.568    3.5519    1.0000    5727.0     0.5505E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 49.951    3.5482    1.0000    5727.0     0.5992E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 49.349    3.5442    1.0000    5727.0     0.6542E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 48.762    3.5399    1.0000    5727.0     0.7183E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 48.188    3.5354    1.0000    5727.0     0.7943E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 47.628    3.5307    1.0000    5727.0     0.8850E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 47.080    3.5257    1.0000    5727.0     0.9927E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 46.545    3.5205    1.0000    5727.0     0.1119E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 46.022    3.5151    1.0000    5727.0     0.1264E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 45.511    3.5095    1.0000    5727.0     0.1428E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 45.011    3.5036    1.0000    5727.0     0.1610E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 44.522    3.4976    1.0000    5727.0     0.1810E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 44.043    3.4915    1.0000    5727.0     0.2025E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 43.574    3.4851    1.0000    5727.0     0.2255E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 43.116    3.4786    1.0000    5727.0     0.2497E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 42.667    3.4719    1.0000    5727.0     0.2749E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 42.227    3.4651    1.0000    5727.0     0.3011E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 41.796    3.4582    1.0000    5727.0     0.3279E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 41.374    3.4511    1.0000    5727.0     0.3552E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 40.960    3.4439    1.0000    5727.0     0.3829E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 40.554    3.4366    1.0000    5727.0     0.4108E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 40.157    3.4291    1.0000    5727.0     0.4388E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 39.767    3.4216    1.0000    5727.0     0.4668E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 39.385    3.4140    1.0000    5727.0     0.4946E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 39.010    3.4063    1.0000    5727.0     0.5222E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 38.642    3.3985    1.0000    5727.0     0.5496E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 38.280    3.3907    1.0000    5727.0     0.5766E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 37.926    3.3828    1.0000    5727.0     0.6032E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 37.578    3.3748    1.0000    5727.0     0.6294E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 37.236    3.3668    1.0000    5727.0     0.6552E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 36.901    3.3588    1.0000    5727.0     0.6804E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 36.571    3.3507    1.0000    5727.0     0.7051E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 36.248    3.3426    1.0000    5727.0     0.7292E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 35.930    3.3345    1.0000    5727.0     0.7526E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 35.617    3.3264    1.0000    5727.0     0.7754E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 35.310    3.3182    1.0000    5727.0     0.7974E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 35.009    3.3101    1.0000    5727.0     0.8185E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 34.712    3.3020    1.0000    5727.0     0.8387E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 34.420    3.2938    1.0000    5727.0     0.8579E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 34.133    3.2857    1.0000    5727.0     0.8760E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 33.851    3.2776    1.0000    5727.0     0.8929E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 33.574    3.2696    1.0000    5727.0     0.9085E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 33.301    3.2615    1.0000    5727.0     0.9227E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 33.032    3.2536    1.0000    5727.0     0.9355E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 32.768    3.2456    1.0000    5727.0     0.9467E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 32.508    3.2377    1.0000    5727.0     0.9565E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 32.252    3.2299    1.0000    5727.0     0.9647E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 32.000    3.2221    1.0000    5727.0     0.9715E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 31.752    3.2144    1.0000    5727.0     0.9770E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 31.508    3.2067    1.0000    5727.0     0.9812E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 31.267    3.1992    1.0000    5727.0     0.9845E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 31.030    3.1917    1.0000    5727.0     0.9870E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 30.797    3.1843    1.0000    5727.0     0.9890E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 30.567    3.1769    1.0000    5727.0     0.9908E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 30.341    3.1697    1.0000    5727.0     0.9926E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 30.118    3.1626    1.0000    5727.0     0.9945E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 29.898    3.1555    1.0000    5727.0     0.9969E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 29.681    3.1486    1.0000    5727.0     0.9997E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 29.468    3.1418    1.0000    5727.0     0.1003E-04   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 29.257    3.1351    1.0000    5727.0     0.1007E-04   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 29.050    3.1285    1.0000    5727.0     0.1011E-04   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 28.845    3.1220    1.0000    5727.0     0.1014E-04   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 28.643    3.1156    1.0000    5727.0     0.1018E-04   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 28.444    3.1094    1.0000    5727.0     0.1020E-04   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 28.248    3.1033    1.0000    5727.0     0.1021E-04   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 28.055    3.0974    1.0000    5727.0     0.1020E-04   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 27.864    3.0915    1.0000    5727.0     0.1016E-04   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 27.676    3.0859    1.0000    5727.0     0.1010E-04   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 27.490    3.0803    1.0000    5727.0     0.1001E-04   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 27.307    3.0749    1.0000    5727.0     0.9877E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 27.126    3.0697    1.0000    5727.0     0.9708E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 26.947    3.0646    1.0000    5727.0     0.9500E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 26.771    3.0597    1.0000    5727.0     0.9250E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 26.597    3.0550    1.0000    5727.0     0.8961E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 26.426    3.0504    1.0000    5727.0     0.8633E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 26.256    3.0460    1.0000    5727.0     0.8268E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 26.089    3.0417    1.0000    5727.0     0.7870E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 25.924    3.0377    1.0000    5727.0     0.7442E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 25.761    3.0338    1.0000    5727.0     0.6988E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 25.600    3.0301    1.0000    5727.0     0.6514E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 25.441    3.0266    1.0000    5727.0     0.6025E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 25.284    3.0233    1.0000    5727.0     0.5527E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 25.129    3.0201    1.0000    5727.0     0.5025E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 85.000    3.4814    1.0000    5727.0     0.6977E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 80.000    3.5060    1.0000    5727.0     0.8747E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 75.000    3.5291    1.0000    5727.0     0.1015E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 70.000    3.5494    1.0000    5727.0     0.1059E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 65.000    3.5649    1.0000    5727.0     0.9409E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 60.000    3.5728    1.0000    5727.0     0.6345E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 55.000    3.5692    1.0000    5727.0     0.3592E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 50.000    3.5485    1.0000    5727.0     0.5953E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 48.000    3.5338    1.0000    5727.0     0.8248E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 46.000    3.5148    1.0000    5727.0     0.1271E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 44.000    3.4909    1.0000    5727.0     0.2046E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 42.000    3.4615    1.0000    5727.0     0.3152E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 40.000    3.4261    1.0000    5727.0     0.4501E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 38.000    3.3845    1.0000    5727.0     0.5977E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 36.000    3.3363    1.0000    5727.0     0.7474E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 34.000    3.2819    1.0000    5727.0     0.8840E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 32.000    3.2221    1.0000    5727.0     0.9715E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 30.000    3.1588    1.0000    5727.0     0.9958E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 29.000    3.1269    1.0000    5727.0     0.1012E-04   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 28.000    3.0957    1.0000    5727.0     0.1019E-04   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 27.000    3.0661    1.0000    5727.0     0.9561E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T  0 26.000    3.0395    1.0000    5727.0     0.7639E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
