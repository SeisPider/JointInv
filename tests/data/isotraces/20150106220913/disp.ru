85.000 3.4880
80.000 3.5143
75.000 3.5388
70.000 3.5600
65.000 3.5757
60.000 3.5829
55.000 3.5773
50.000 3.5532
48.000 3.5368
46.000 3.5159
44.000 3.4899
42.000 3.4584
40.000 3.4211
38.000 3.3778
36.000 3.3287
34.000 3.2746
32.000 3.2169
30.000 3.1587
29.000 3.1309
28.000 3.1054
27.000 3.0833
26.000 3.0667
