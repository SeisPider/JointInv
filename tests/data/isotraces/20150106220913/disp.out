SURF96 R U T  0 85.000    3.4814    1.0000    
SURF96 R C T  0 85.000    6.5183    1.0000    
SURF96 R U T  0 80.000    3.5060    1.0000    
SURF96 R C T  0 80.000    6.2024    1.0000    
SURF96 R U T  0 75.000    3.5291    1.0000    
SURF96 R C T  0 75.000    5.9201    1.0000    
SURF96 R U T  0 70.000    3.5494    1.0000    
SURF96 R C T  0 70.000    5.6660    1.0000    
SURF96 R U T  0 65.000    3.5649    1.0000    
SURF96 R C T  0 65.000    5.4358    1.0000    
SURF96 R U T  0 60.000    3.5728    1.0000    
SURF96 R C T  0 60.000    5.2257    1.0000    
SURF96 R U T  0 55.000    3.5692    1.0000    
SURF96 R C T  0 55.000    5.0316    1.0000    
SURF96 R U T  0 50.000    3.5485    1.0000    
SURF96 R C T  0 50.000    4.8494    1.0000    
SURF96 R U T  0 48.000    3.5338    1.0000    
SURF96 R C T  0 48.000    4.7788    1.0000    
SURF96 R U T  0 46.000    3.5148    1.0000    
SURF96 R C T  0 46.000    4.7090    1.0000    
SURF96 R U T  0 44.000    3.4909    1.0000    
SURF96 R C T  0 44.000    4.6395    1.0000    
SURF96 R U T  0 42.000    3.4615    1.0000    
SURF96 R C T  0 42.000    4.5700    1.0000    
SURF96 R U T  0 40.000    3.4261    1.0000    
SURF96 R C T  0 40.000    4.5000    1.0000    
SURF96 R U T  0 38.000    3.3845    1.0000    
SURF96 R C T  0 38.000    4.4288    1.0000    
SURF96 R U T  0 36.000    3.3363    1.0000    
SURF96 R C T  0 36.000    4.3559    1.0000    
SURF96 R U T  0 34.000    3.2819    1.0000    
SURF96 R C T  0 34.000    4.2807    1.0000    
SURF96 R U T  0 32.000    3.2221    1.0000    
SURF96 R C T  0 32.000    4.2025    1.0000    
SURF96 R U T  0 30.000    3.1588    1.0000    
SURF96 R C T  0 30.000    4.1207    1.0000    
SURF96 R U T  0 29.000    3.1269    1.0000    
SURF96 R C T  0 29.000    4.0784    1.0000    
SURF96 R U T  0 28.000    3.0957    1.0000    
SURF96 R C T  0 28.000    4.0351    1.0000    
SURF96 R U T  0 27.000    3.0661    1.0000    
SURF96 R C T  0 27.000    3.9909    1.0000    
SURF96 R U T  0 26.000    3.0395    1.0000    
SURF96 R C T  0 26.000    3.9460    1.0000    
